openedx_external_enrollments/edxapp_wrapper/backends/openedx_permissions_i_v1.py

```diff
@@ -1,6 +1,5 @@
 """ Backend abstraction """
-from openedx.core.lib.api.permissions import (
-    # pylint: disable=unused-import
+from openedx.core.lib.api.permissions import (  # pylint: disable=unused-import, import-error
```

Comment:
Is the unused-import necessary? @andrey-canon

Reply:
pylint says yes

Reply:
Ummm, ok.